app/models/manager_refresh/save_collection/saver/base.rb

```diff
@@ -67,6 +67,10 @@ def save!(inventory_collection, association)
           end
         end
 
+        unless inventory_collection.custom_reconnect_block.nil?
+          inventory_collection.custom_reconnect_block.call(inventory_collection, inventory_objects_index, attributes_index)
+        end
```

Comment:
for now we need to remove them from inventory_objects_index, the rest will get created.

I think once we implement this for hosts and vms, we can generalize this alghorithm